### ISSUE TITLE
Both my caddx and vista loose root, WTFOS, and OSD

### ISSUE THREAD

**Issue by Spit29** [fpv-wtf/wtfos-configurator#192]
Hello there, this has happened to me 2x now where my Caddx and Vista randomly lose OSD and their root. It works perfectly for multiple batteries and then suddenly says "WAITING FOR OSD" at the bottom right corner. The goggles still seem to have WTFOS after the error but the caddx seems to lose it. If I try another drone that has WTFOS it will also start giving the "WAITING FOR OSD" warning on that one as well. Here is the error when connecting the caddx and air units afterwards (see picture)
![Captureerere](https://user-images.githubusercontent.com/114850284/194767998-826cb6d3-31e4-454b-bd53-b94c595ac161.JPG)

**Comment by stylesuxx** [comment]
You had a slot switch, that might happen if you rapidly unplug/re-plug power, the air-unit thinks the slot is broken and will boot from the other slot.

Not really a configurator issue and not much you can do about it right now - apart from running the fix.

**Comment by Spit29** [comment]
So just make sure not to unplug and re-plug rapidly? 
What was strange then is when I tested on another drone it happened as well, and I only powered up the air unit 1 time.

**Comment by stylesuxx** [comment]
Yes, unplugging and re-plugging rapidly are the main culprits of this issue. Hard crashes can also lead to this issue, as do spikes/dips in supply voltage.

It happening from plugging in only one time sounds strange to me, but it is a bit of a protective mechanism of the air units: If it thinks something is wrong, it will switch slots. Right now there is nothing you can do apart from switching back manually, but we are working on a long term solution for this issue.

**Comment by Spit29** [comment]
Sounds good, thanks for the reply!
